### PR TITLE
IBX-5981: Fixed handling multiple custom classes

### PR DESCRIPTION
**Pull request by dew326**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | https://issues.ibexa.co/browse/IBX-5981
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | 4.5
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

**Requires:** https://github.com/ibexa/admin-ui/pull/828

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

**Review by Gengar-i — APPROVED** [review]

**Review by lucasOsti — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

File: src/bundle/Resources/public/js/CKEditor/custom-attributes/ui/custom-attributes-form-view.js [hunk 1]
```diff
@@ -209,9 +209,8 @@ class IbexaCustomAttributesFormView extends View {
         addListToDropdown(labeledDropdown.fieldView, itemsList);
 
         this.listenTo(labeledDropdown.fieldView, 'execute', (event) => {
-            const value = config.multiple
-                ? [...new Set([...labeledDropdown.fieldView.element.value.split(','), event.source.value])].join(',')
-                : event.source.value;
+            const alreadySelectedValues = labeledDropdown.fieldView.element.value ? labeledDropdown.fieldView.element.value.split(',') : [];
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/CKEditor/custom-attributes/ui/custom-attributes-form-view.js:212]: maybe we can use optional chaining?
```suggestion
            const alreadySelectedValues = labeledDropdown.fieldView.element.value.split?.(',') ?? [];
```

**Review by tischsoic — APPROVED** [review]

**Review by micszo — COMMENTED** [review]
There is a visual issue without default value before choosing first value.

Screenshot 2023-06-22 at 15 34 12

**Comment by micszo** [comment]
When using following config 
```
                        paragraph:
                            choices: [ regular, special, tip_box, warning_box ]
#                            default_value: regular
                            required: false
                            multiple: false
                        heading:
                            choices: [ foo, bar ]
#                            default_value: foo
                            required: false
                            multiple: true
                        heading2:
                            choices: [ titre1, titre2, titre3 ]
#                            default_value: titre1
                            required: true
                            multiple: false
                        heading3:
                            choices: [ test1, test2, test3 ]
#                            default_value: test1
                            required: true
                            multiple: true
```

values are being carried over between classes:

Screenshot 2023-06-22 at 15 48 21

**Comment by micszo** [comment]
Console error occurs when saving custom attributes:

Screenshot 2023-06-22 at 15 41 18

**Review by micszo — CHANGES_REQUESTED** [review]
Please see comments above.

**Comment by micszo** [comment]
Additional feedback from customer: https://jsd.ez.no/browse/CS-11138?focusedCommentId=276759&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-276759

**Comment by dew326** [comment]
@micszo all issues should be fixed. Please note there is another PR https://github.com/ibexa/admin-ui/pull/828

**Review by micszo — APPROVED** [review]
Thanks for fixing all the issues!

QA Approved on Ibexa Experience 4.5.1-dev.

**Pull request merged**